core/iwasm/aot/aot_loader.c

```diff
@@ -10,6 +10,7 @@
 #include "../common/wasm_runtime_common.h"
 #include "../common/wasm_native.h"
 #include "../compilation/aot.h"
+#include <stdint.h>
```

Comment:
Had better remove this line, the platform_internal.h should already include related header file?